packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -104,12 +124,21 @@ class DatePickerInput extends React.Component {
     }
   }
 
+  handleOnInputFocus(event) {
+    if (this.props.onInputFocus) {
```

Comment:
This wrapper feels extraneous, as your onInputFocus is all that's done on focus. Can you remove this extraneous wrapper and just pass this.props.onInputFocus to the element?

Reply:
Updated. https://github.com/cerner/terra-core/pull/974/commits/f55edc6bf3657e8b9740cd65f5c159af25b21eb1